inc/vcf/fixer.hpp

```diff
@@ -54,13 +56,29 @@ namespace ebi
         virtual void visit(FileformatError &error) override;
         virtual void visit(ChromosomeBodyError &error) override;
         virtual void visit(PositionBodyError &error) override;
+
+        /**
+         * fix: remove duplicate ids - keep the first one and remove consequent ones
+         */
         virtual void visit(IdBodyError &error) override;
         virtual void visit(ReferenceAlleleBodyError &error) override;
         virtual void visit(AlternateAllelesBodyError &error) override;
         virtual void visit(QualityBodyError &error) override;
+
+        /**
+         * fix:
+         * - if any filter string is 0, remove it
+         * - remove duplicate filter strings - keep the first and remove consequent ones
+         */
         virtual void visit(FilterBodyError &error) override;
+
         /**
-         * fix: remove the info field that caused the error
+         * fix:
+         * - fix duplicates in info field :
+         *     - if even one value differs, remove all the duplicate key fields
+         *     - else, keep one duplicate field and remove all the others
```

Comment:
Yes, they'll be kept only if all the values match for the field match. 
e.g. : A=1;A=1;B=2;A=1 gets fixed to A=1;B=2
But A=1;A=2;B=2 gets fixed to B=2

But I guess the comment in the code is not clear? I will update it.